### ISSUE TITLE
Why cannot we repeat the training result of ACC=0.987 using CASIA?

### ISSUE THREAD

**Issue by ZvcXoyo** [davidsandberg/facenet#689]
Hello David and anyone see here,
    Some friends and I recently tried to train some models from scratch(such as center loss, AM, arcface,etc). Firstly, we decide to repeat the result of 20170511-185253 which you supplied, and we tried the settings as you recommended with the same CASIA and LFW(Face alignment using your align_dataset_mtcnn.py). 
```
logs_base_dir: ~/PycharmProjects/ExtractorTrain/logs/centerloss/
models_base_dir: ~/PycharmProjects/ExtractorTrain/models/centerloss/
gpu_memory_fraction: 0.9
pretrained_model: None
data_dir: /media/ccf-dl-1/00012641000200F0/Dataset/CASIA-WebFace/train/result-Clean182/crop_image
model_def: models.inception_resnet_v1
max_nrof_epochs: 80
batch_size: 90
image_size: 160
epoch_size: 1000
embedding_size: 128
random_crop: True
random_flip: True
random_rotate: False
keep_probability: 0.8
weight_decay: 5e-05
center_loss_factor: 0.01
center_loss_alfa: 0.9
optimizer: ADAM
learning_rate: -1.0
learning_rate_decay_epochs: 100
learning_rate_decay_factor: 1.0
moving_average_decay: 0.9999
seed: 666
nrof_preprocess_threads: 4
log_histograms: False
learning_rate_schedule_file: /home/ccf-dl-1/facenet/data/learning_rate_schedule_classifier_casia.txt
filter_filename: 
filter_percentile: 100.0
filter_min_nrof_images_per_class: 0
lfw_pairs: /home/ccf-dl-1/facenet/data/pairs.txt
lfw_file_ext: jpg
lfw_dir: /media/ccf-dl-1/00012641000200F0/Dataset/LFW/train/result/crop_image
lfw_batch_size: 100
lfw_nrof_folds: 10
```

However,we could got the result of 97%(ACC) as follows:

```
Accuracy: 0.979+-0.005
Validation rate: 0.88833+-0.02267 @ FAR=0.00133
Area Under Curve (AUC): 0.998
Equal Error Rate (EER): 0.019
```
***
    After this, we three also tried different alignment methods, batch size, bigger epoch num and VGGFACE2 dataset, but got similar results(97~98). 

    So I want to know how to do to get the result 99%+?
    Thanks!

**Comment by ZvcXoyo** [comment]
@Shahnawazgrewal

**Comment by xxllp** [comment]
你训练了多少epoch

**Comment by ZvcXoyo** [comment]
@xxllp  80\120还有最新的276 epcoh都试过, 也试过把学习率调小 发现效果都没有达到像作者这么好的,你复现过么?用的什么设置能发一下看看么

**Comment by xxllp** [comment]
我还在跑，看到结果再说，调参是个苦力活。

**Comment by ZvcXoyo** [comment]
@xxllp  好的 期待你的结果

**Comment by xxllp** [comment]
@ZvcXoyo ，训练到一半掉了怎么继续啊

**Comment by ZvcXoyo** [comment]
有个—pretrained_model参数 把这个参数定位到你挂掉的模型，再次启动训练就会先读取上次的权重，他会自动生成新的模型文件夹，按当前时间命名，不会和原模型冲突。当然，由于epoch还是从头计数，所以如果你用了学习率的计划表，那么注意修改到合适的值。

发送自 Windows 10 版邮件应用

发件人: potter
发送时间: 2018年4月27日 13:13
收件人: davidsandberg/facenet
抄送: ZvcXoyo; Mention
主题: Re: [davidsandberg/facenet] Why cannot we repeat the training resultof ACC=0.987 using CASIA? (#689)

@ZvcXoyo ，训练到一半掉了怎么继续啊

**Comment by xxllp** [comment]
我改好了，这个。但是才跑到lfw 0.97+，这个收敛太慢了

**Comment by ZvcXoyo** [comment]
我到现在也没弄明白作者是怎么做到原始的softmax都能跑到99.65%的……

**Comment by xxllp** [comment]
没有就没有呗，说不定作者留了一手

**Comment by ZvcXoyo** [comment]
彳亍口巴

**Comment by xxllp** [comment]
能加个qq 聊下细节吗：我的505279206

**Comment by LaviLiu** [comment]
同样，训练了MS_celeb_1m和vggface2两个数据集，效果都不是很好，一直在这附近晃悠，改了batchsize, 学习率都没效果
Accuracy: 0.98350+-0.00669
Validation rate: 0.84500+-0.02818 @ FAR=0.00100
可以有人给指导一下吗？

**Comment by phoebushe** [comment]
@Laviyy 我的VGGFACE2出来99.4X%，一直上不了99.5%，作者99.66%怎么训练的。。

**Comment by Tomhouxin** [comment]
@phoebushe ，你所说的99.4%，是你自己重新训练的vggface2数据集？，然后测试用的lfw？

**Comment by phoebushe** [comment]
@Tomhouxin hi，是的

**Comment by tangchen2** [comment]
> @Tomhouxin hi，是的
请问 你是用softmax loss来训练的嘛，后期有没有用triplet loss进行finetune呢，我这边好像遇到了一些问题，只有softmax能够训练出不错的结果
